pkg/apis/autoscaling/v1alpha1/kpa_types.go

```diff
@@ -164,8 +172,6 @@ func (rs *PodAutoscalerStatus) MarkInactive(reason, message string) {
 	podCondSet.Manage(rs).MarkFalse(PodAutoscalerConditionActive, reason, message)
 }
 
-// CanScaleToZero checks whether the pod autoscaler has been in an inactive state
-// for at least the specified grace period.
 func (rs *PodAutoscalerStatus) CanScaleToZero(gracePeriod time.Duration) bool {
```

Comment:
Nit: why did you remove this godoc comment?  I think that any exported function should have a comment.  See also `CanMarkInactive` below.

Reply:
Whoops, will fix!